### DISCUSSION TITLE
API Documentation Request for MeshCentral Integration with Ticket Management Tool

### DISCUSSION THREAD

**Discussion by Givannixs** [Ylianst/MeshCentral#6258] [Q&A]
Hi everyone,

I currently use Tactical RMM with MeshCentral to manage my devices and remote access. I would like to integrate the remote access functionalities of MeshCentral with another tool I use to manage my tickets. Is there any API documentation that can assist me with this integration?

Thank you in advance!

**Comment by si458** [comment]
yes and no, you can use the embed option to embed a view of meshcentral into your web ui if that helps?
https://ylianst.github.io/MeshCentral/meshcentral/#embedding-meshcentral

**Comment by chrios** [comment]
I've been working on this, and the only solution i've found is to directly query the database for information on the nodes to sync into Requesttracker assets, then also, you can use the Integration field in Custom Field for request tracker, to make a clickable link to meshcentral for remote control. But the challenge i am having is that Request tracker URL encodes the ID field, so the clickable link is broken. Food for thought for others attempting this.
